### PR TITLE
update(git-semver-tags): v4.1 update and other minor changes

### PR DESCRIPTION
**Pull request by peterblazejewicz**
- `skipUnstable` option
- definition cleanup
- configuration cleanup
- version bump
- maintainer added

https://github.com/conventional-changelog/conventional-changelog/blob/master/packages/git-semver-tags/CHANGELOG.md

Thanks!

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

File: types/git-semver-tags/git-semver-tags-tests.ts [hunk 1]
```diff
@@ -1,13 +1,17 @@
-"use strict";
-
-import gitSemverTags from "git-semver-tags";
+import gitSemverTags = require('git-semver-tags');
```
Comment (JasonHK) [inline types/git-semver-tags/git-semver-tags-tests.ts:1]: Will this be better?

```suggestion
import * as gitSemverTags from "git-semver-tags";
```

File: types/git-semver-tags/index.d.ts [hunk 2]
```diff
@@ -26,6 +29,11 @@ declare namespace gitSemverTags {
          * Specify a prefix for the git tag to be ignored from the semver checks.
          */
         tagPrefix?: string;
+
+        /**
+         * If given, unstable tags will be skipped [ e.g. x.x.x-alpha.1, x.x.x-rc.2 ]
```
Comment (JasonHK) [inline types/git-semver-tags/index.d.ts:34]:
```suggestion
         * If given, unstable tags (e.g. `x.x.x-alpha.1`, `x.x.x-rc.2`) will be skipped.
```

**Review by JasonHK — CHANGES_REQUESTED** [review]
Just some tweaks needed, thanks! :smile:

**Review by JasonHK — APPROVED** [review]
Thanks!

**Comment by peterblazejewicz** [comment]
Ready to merge 💋

**Pull request merged**